deploy/olm-catalog/nxiq-operator-certified/1.90.0-1/nxiq-operator-certified.v1.90.0-1.clusterserviceversion.yaml

```diff
@@ -145,7 +145,7 @@ spec:
 
     | Parameter            | Description                                                  | Default           |
     | -------------------- | ------------------------------------------------------------ | ----------------- |
-    | `iq.imageName`       | The image name to use for the IQ Container, eg `sonatype/nexus-iq-server`  | `"registry.connect.redhat.com/sonatype/nexus-iq-server"`              |
+    | `iq.imageName`       | The image name to use for the IQ Container, eg `sonatype/nexus-iq-server`  | `"registry.connect.redhat.com/sonatype/nexus-iq-server:1.90.0-1"`              |
```

Comment:
is this value going to be maintained as we release new versions? It might be safer to replace that with a more generic placeholder

Reply:
since all the defaults are visible in the yaml as you're creating an instance, how would you feel about losing the default value column in this description?  Each new version of this CSV yaml will specify the new application image, so anything but the value might be misleading. 🤷

Reply:
Possibly we could pull the "default" into the "description" and preface it with "for example: ...". It seems like that's the only real benefit here. If we do that for a few key places (and I think we're _already_ doing that), then the default value column can be removed.

TBH, I'm not sure they are 'defaults' anyway. So removing that column could be a real plus.